packages/ui/src/components/vuestic-components/va-expand/VaExpand.demo.vue

```diff
@@ -6,7 +6,7 @@
         style="width: 400px;"
         :header="expandHeader"
       >
-        <div>
+        <div style="padding: 1rem;">
```

Comment:
You can move these to scoped styles just to keep demo a bit cleaner.
(I know I said "inline styles are best" before, but that opinion changed :) )

Reply:
Should i remove it from everywhere or just this demo?

Reply:
@asvae up to you. It's not a big deal.